doc/releases.md

```diff
@@ -91,7 +96,7 @@ The _CHANGELOG.md_ entry should take the following form:
 * Include the full list of commits since the last release here
 ```
 
-### 5. Update _src/node_version.h_
+### 4. Update _src/node_version.h_
```

Comment:
agreed, and I messed up the tag comment for 2.3.3 today as well, this could either be Jenkinsised or put into a script but I don't see an obvious path yet